### PR TITLE
disable mixed usage of cypher & ngql

### PR DESCRIPTION
**Pull request by nevermore3**
#### What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### What does this PR do?

disable mixed usesage of cypher & ngql

#### Which issue(s)/PR(s) this PR relates to?

close https://github.com/vesoft-inc/nebula/issues/3477
  
#### Checklist:
- [x] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Review by yixinglu — DISMISSED** [review]
LGTM

**Comment by Shylock-Hg** [comment]
I think disable output to open cyher is enough, like #2537

**Comment by CPWstatic** [comment]
> I think disable output to open cyher is enough, like #2537

It's a great job. But there would have a more compatible implementation of `match`, so we could disable all of these usages now.

**Review by CPWstatic — APPROVED** [review]

File: tests/tck/features/bugfix/MatchUsedInPipe.feature [hunk 1]
```diff
@@ -75,14 +77,11 @@ Feature: Test match used in pipe
...
   Scenario: Set op after match
     When executing query:
       """
       MATCH (n:player{name:"Tim Duncan"}) RETURN n UNION MATCH (n:player{name:"Tony Parker"}) RETURN n
       """
-    Then the result should be, in any order, with relax comparison:
-      | n               |
-      | ("Tim Duncan")  |
-      | ("Tony Parker") |
+    Then a SyntaxError should be raised at runtime: syntax error near `UNION'
```
Comment (CPWstatic) [inline tests/tck/features/bugfix/MatchUsedInPipe.feature:87]: This is not right, MATCH ... UNION MATCH ... is a legal cypher syntax.

Comment (nevermore3) [reply tests/tck/features/bugfix/MatchUsedInPipe.feature]: done

**Review by CPWstatic — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Comment by cooper-lzy** [comment]
image

**Pull request merged**